Userland/Libraries/LibJS/Runtime/Temporal/CalendarPrototype.cpp

```diff
@@ -219,6 +219,8 @@ JS_DEFINE_NATIVE_FUNCTION(CalendarPrototype::date_add)
     if (vm.exception())
         return {};
 
+    // TODO: Use BalanceDuration before AddISODate, see https://github.com/tc39/proposal-temporal/commit/6178ed35400fb56ef4013ce9d0163212ec096e57#diff-1e3b606bd8b3ef5c33eaf7b43afdfc240b0a926c8d0a66e02bfc9e1a7e8de8a2R798-R799
```

Comment:
`BalanceDuration` was added in my latest PR :)

Reply:
hah I didn't feel like implementing that at the time, nice :D